modules/protect/shared-functions.php

```diff
@@ -180,4 +180,41 @@ function jetpack_convert_ip_address( $ip ) {
 		return inet_pton( $ip );
 	}
 	return ip2long( $ip );
+}
+
+/**
+ * Checks that a given IP address is within a given low - high range.
+ * Servers that support inet_pton will use that function to convert the ip to number,
+ * while other servers will use ip2long.
+ *
+ * NOTE: servers that do not support inet_pton cannot support ipv6.
+ *
+ * @param $ip
+ * @param $range_low
+ * @param $range_high
+ *
+ * @return bool
+ */
+function ip_address_is_in_range( $ip, $range_low, $range_high ) {
```

Comment:
I'd recommend namespacing this-- something like `jpp_ip_address_is_in_range`

As is, the function name feels a bit too generic, could run up against something